tests/testsuite/git.rs

```diff
@@ -2787,66 +2787,6 @@ to proceed despite [..]
     git_project.cargo("package --no-verify").run();
 }
 
-#[cargo_test]
-fn default_not_master() {
```

Comment:
Is there a reason to remove these two tests instead of just updating the checks for the new behavior?  It seems like it would be useful to have something to check the behavior in these two cases, and I don't see any other tests that covers it.

Reply:
Sure yeah, I don't mind updating their behavior